domain/network/service/space.go

```diff
@@ -327,7 +328,7 @@ func (s *ProviderSpaces) saveSpaces(ctx context.Context, providerSpaces []networ
 			spaceName := network.ConvertSpaceName(string(spaceInfo.Name), spaceNames)
 
 			s.logger.Debugf("Adding space %s from provider %s", spaceName, string(spaceInfo.ProviderId))
-			spaceID, err := s.spaceService.AddSpace(
```

Comment:
This was the actual offending line, the `spaceID` var is not assigned to the `spaceID` var defined in line 320 :man_facepalming: